src/main/java/Duke/Tasks/Deadline.java

```diff
@@ -0,0 +1,37 @@
+package Duke.Tasks;
+
+public class Deadline extends Task {
+
+    private String endTime;
+
+    public Deadline(String nameInit, String time) {
+        super(nameInit);
+        this.endTime = time;
+    }
+
+    @Override
+    public String toString() {
+        String outputString = "[D]";
```

Comment:
same here... maybe you should consider the use of constants to avoid magic strings ;)